### DISCUSSION TITLE
Bootstrap accordion

### DISCUSSION THREAD

**Discussion by wilcar** [quarto-dev/quarto-cli#6036] [Q&A]
Do you think it's possible to integrate a bootstrap accordion into quarto with callout-like syntax?

https://getbootstrap.com/docs/5.1/components/accordion/

**Comment by mcanouil** [comment]
Bootstrap is included/embedded in Quarto (with few configuration changes), so yes to can do this using either raw HTML or using fenced divs, see <https://quarto.org/docs/authoring/markdown-basics.html#divs-and-spans>.

FYI, the current Bootstrap version embedded is v5.1.3.

PS: I changed the category from "Feature Requests" to "Q&A" which seems more appropriate.

**Reply by mcanouil** [reply]
FYI:
- https://github.com/quarto-dev/quarto-cli/discussions/8828
